content/en/running-programs/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Running CHT implementations"
+linkTitle: "CHT Implementations"
+weight : 2
+---
+
+There are various ways of running Community Health Toolkit implementations. It is important to understand the different models for integrating CHT tools into a community health program as discussed below.
+
+**Accompaniment approach**
+
+In this approach, Medic accompanies partners; including ministries of health, through their digital journey that may include designing, building, deploying and supporting CHT based tools for their unique health system needs. 
+
+**Community led implementations**
+
+This model leverages the robust technical expertise of the CHT community of practice. A network of local and regional technical organizations that have been onboarded on CHT are able to provide design, development, deployment and support services for implementing partners - including ministries of health. This network also contributes their learnings to the community through documentation, reference applications, CHT forum and CHT core improvements. Community led implementations emphasize on the importance of collaboration, local ownership and sustainability.
```

Comment:
```suggestion
Community led implementations emphasize on the importance of collaboration, local ownership and sustainability. This is a self-led model where a community identifies a digital health problem and solves it using CHT tools. The CHT also has a community of practice that is made up of a network of local and regional technical organizations that have been onboarded on CHT and are able to provide design, development, deployment and support services for implementing partners - including ministries of health. This network also contributes their learnings to the community through documentation, reference applications, CHT forum and CHT core improvements. 
```

Reply:
@garethbowen ,is this okay or should I not mention the community of practice and technical organisations completely?

Reply:
I think it's perfect, because you start with self-led and go into the options for support. Thanks!